Metron/.github/ISSUE_TEMPLATE/bug.md

```
@@ -0,0 +1,25 @@
+---
+name: Bug Report
+about: Report a defect
+title: "[BUG] "
+labels: bug
+---
+
+## Describe the bug
+<!-- Clear, concise description -->
+
+## Reproduction steps
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)